docs/source/development.md

```diff
@@ -0,0 +1,90 @@
+# Development
+
+```{mermaid}
+flowchart LR
+    subgraph Dagger
+        containers["Container(s)"]
+    end
+    subgraph Poetry
+        python[Python]
+    end
+    python --> |running within| containers
+```
+
+Pcytominer-transform is primarily written in Python with related environments managed by Python [Poetry](https://python-poetry.org/).
+[Dagger](https://docs.dagger.io/) is used to help create consistent testing locally and with automated tests via containers.
```

Comment:
```suggestion
We use [Dagger](https://docs.dagger.io/) for consistent local testing and for automated tests via containers.
```

Please write in active (instead of passive) voice

Reply:
Thank you for catching these! I'll update with the suggestions and do some editing as well.